scheduler.go

```diff
@@ -228,28 +222,25 @@ func main() {
 		log.Fatal(fmt.Sprintf("Unable to create scheduler driver: %s", err))
 	}
 
-	// If here, then all command-line arguments validate.
-	// Creating logger and attaching different logging platforms.
-	startTime := time.Now()
-	formattedStartTime := startTime.Format("20060102150405")
 	// Checking if prefix contains any special characters.
 	if strings.Contains(*pcplogPrefix, "/") {
 		log.Fatal("log file prefix should not contain '/'.")
 	}
-	logPrefix := *pcplogPrefix + "_" + formattedStartTime
-	logger := elekLogDef.BuildLogger(startTime, logPrefix)
-	// Starting the logging go-routine.
-	go logger.Listen(logMType, logMsg)
+
+	// Build Logger.
+	if err := elekLog.BuildLogger(*pcplogPrefix, *logConfigFilename); err != nil {
```

Comment:
In the future, we can have the log prefix to be part of the log config.

Reply:
Oh, okay.